src/Views/GameController/GameController.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FunctionComponent } from "react";
+import { robotSimulatorSlice } from "../../RobotSimulator/robotSimulatorSlice";
+import { useDispatch } from "react-redux";
+
+/**
+ * Component for the game controller
+ */
+export const GameController: FunctionComponent = () => {
+  const dispatch = useDispatch();
+  // TODO instead of buttons use the buttons from the control-cross but this path has to be split first.
+  function onBlueButtonClicked(event: React.MouseEvent) {
```

Comment:
Consolidate into one handler please.

Reply:
I added a service method "handleMotorChange" to control left and right motor at once.
But I don't like to add business code (like handleMotorChange) inside the SVG tags and rather want to stay with seperating UI callbacks and business code.